contracts/test/CompoundGovernorSetQuorum.t.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.26;
+
+import {ProposalTest} from "contracts/test/helpers/ProposalTest.sol";
+
+contract CompoundGovernorInitializeTest is ProposalTest {
+    function _submitPassAndExecuteProposalToSetNewQuorum(address _proposer, uint256 _amount) public {
+        address[] memory _targets = new address[](1);
+        _targets[0] = address(governor);
+
+        uint256[] memory _values = new uint256[](1);
+        _values[0] = 0;
+
+        bytes[] memory _calldatas = new bytes[](1);
+        _calldatas[0] = _buildProposalData("setQuorum(uint256)", abi.encode(_amount));
+
+        ConstructedProposal memory _proposal = ConstructedProposal(_targets, _values, _calldatas, "Set New Quorum");
+        _submitPassQueueAndExecuteProposal(_proposer, _proposal);
+    }
+
+    function _submitAndFailProposalToSetNewQuorum(address _proposer, uint256 _amount) public {
+        address[] memory _targets = new address[](1);
+        _targets[0] = address(governor);
+
+        uint256[] memory _values = new uint256[](1);
+        _values[0] = 0;
+
+        bytes[] memory _calldatas = new bytes[](1);
+        _calldatas[0] = _buildProposalData("setQuorum(uint256)", abi.encode(_amount));
+
+        ConstructedProposal memory _proposal = ConstructedProposal(_targets, _values, _calldatas, "Set New Quorum");
+        _submitAndFailProposal(_proposer, _proposal);
+    }
+
+    function testFuzz_SetQuorum(uint256 _newQuorum) public {
+        _newQuorum = uint224(bound(_newQuorum, 1, 1000));
```

Comment:
why are we casting to `uint224`?

Reply:
Left over typo from when branched off of PR that implemented settable quorum before datatypes were changed.
Fixed: dcc2f60ecbd5cd86e72abb45c1d7e23f7949a557